package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@conflux-dev/contracts",
+  "name": "@confluxfans/solidity",
```

Comment:
How about to keep the same name convention with openzeppelin? E.g. `@confluxfans/contracts`, which is used in the package.json

Reply:
There is a new package.json in contracts folder, which is only used for publishing npm package